api/CanvasPattern.json

```diff
@@ -5,7 +5,7 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/CanvasPattern",
         "support": {
           "chrome": {
-            "version_added": "4"
+            "version_added": "6"
```

Comment:
This ought to match the data at https://github.com/mdn/browser-compat-data/blob/f3d419658370750830c5e8450922cba17b311000/api/CanvasRenderingContext2D.json#L1029, which says Chrome 1. Maybe the interface object just wasn't exposed originally?

Reply:
Most likely so -- I'll reopen this PR with a custom test introduced in collector v1.1.9!